packages/app/src/routers/ViewRouter/ViewRouter.tsx

```diff
@@ -57,15 +51,14 @@ const View = (props: ViewProps) => {
   });
   const theme = useThemeWatcher();
   const changesetId = props.versionId ? fetchedVersion?.changesetId : undefined;
-  return state || !props.versionId ? (
+  return (state || !props.versionId) && AuthClient.client ? (
     <Viewer
       changeSetId={changesetId}
-      contextId={props.projectId ?? ""}
+      iTwinId={props.projectId ?? ""}
       iModelId={props.iModelId ?? ""}
-      authConfig={{ oidcClient: AuthClient.client }}
+      authClient={AuthClient.client}
       theme={theme}
-      backend={{ buddiRegion }}
-      uiProviders={[new SimpleBgMapToggleProvider()]}
+      enablePerformanceMonitors={false}
```

Comment:
you should prob set. this to true, all our apps should have this enabled

Reply:
What is it doing ? We likely dont look at appInsights, I wasnt sure if we wanted to have that enabled on this demo stuff.

Reply:
Ok, I'll switch.

Reply:
Enabled the performanceMonitors